### ISSUE TITLE
Orchagent receives neighbor DEL operation when the reference count of the neighbor is not zero after reboot

### ISSUE THREAD

**Issue by shi-su** [Azure/sonic-swss#1483]
**Description**
The orchagent receives neighbor DEL operation when the reference count of the neighbor is not zero with ping traffic after reboot (cold/fast/warm).

**Steps to reproduce the issue**
Run warm/fast-reboot test (advanced-reboot). Confirmed the issue also happens with cold reboot with the same ping traffic of warm/fast-reboot test.

**Describe the results you received**
Orchagent receives neighbor DEL operation. The operation fails due to a non-zero reference count.
```
Oct 26 22:23:15.259716 vlab-01 INFO swss#orchagent: :- removeNeighbor: Failed to remove still referenced neighbor 1e:92:d0:9e:6d:64 on PortChannel0002
Oct 26 22:23:15.259851 vlab-01 INFO swss#orchagent: :- removeNeighbor: Failed to remove still referenced neighbor 26:74:37:ff:3b:fa on PortChannel0004
```

**Describe the results you received**
Expect not to receive neighbor deletion operations with reference counter.

**Comment by shi-su** [comment]
@qiluo-msft @lguohan @zhenggen-xu for visibility

**Comment by prsunny** [comment]
Looks like related to this  - https://github.com/Azure/sonic-buildimage/issues/4400#issuecomment-714814839

**Comment by shi-su** [comment]
> Looks like related to this - [Azure/sonic-buildimage#4400 (comment)](https://github.com/Azure/sonic-buildimage/issues/4400#issuecomment-714814839)

Yes, I just submitted #1485  to resolve it for neighorch and fix https://github.com/Azure/sonic-buildimage/issues/4400. While the source of the DEL operation remains unknown, this issue aims to track the source of the unexpected operation
